gdnative-core/src/init/property/hint.rs

```diff
@@ -231,12 +231,12 @@ impl ExpEasingHint {
     pub fn to_godot_hint_string(&self) -> GodotString {
         let mut s = String::new();
 
-        let mut is_second = false;
-
-        if self.is_attenuation {
+        let is_second = if self.is_attenuation {
             s.push_str("attenuation");
-            is_second = true;
-        }
+            true
+        } else {
+            false
+        };
```

Comment:
This one should be silenced instead of blindly followed. It should be clear from the name `is_second`, that it's used to check whether a comma needs to be inserted before the next component. This transformation just *happens* to work because there are currently only two components. If you don't like silencing the warning, you may:

- Refactor this entire method into something else that don't cause any warnings.
- ...or, change L242 to read `if self.is_attenuation {`, removing the need for an `is_second` binding.